File: 백준/Bronze/4344. 평균은 넘겠지/평균은 넘겠지.py
import sys

C = int(sys.stdin.readline())
test_cases = []
for i in range(C):
    test_cases.append(list(map(int, sys.stdin.readline().split())))

for test_case in test_cases:
    n = test_case[0]
    test_case = test_case[1:]

    avg = sum(test_case) / n
    above_avg_count = 0

    for score in test_case:
        if score > avg:
            above_avg_count += 1

    result = (above_avg_count / n) * 100

    print("{:.3f}%".format(result))

File: 백준/Bronze/4344. 평균은 넘겠지/평균은 넘겠지.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int c = Integer.parseInt(br.readLine());

        for (int i = 0; i < c; i++) {
            StringTokenizer st = new StringTokenizer(br.readLine());
            int n = Integer.parseInt(st.nextToken());
            int[] testCase = new int[n];

            int sum = 0;
            for (int j = 0; j < n; j++){
                testCase[j] = Integer.parseInt(st.nextToken());
                sum += testCase[j];
            }

            double avg = (double) sum / n;
            int aboveAvgCount = 0;

            for (int score : testCase) {
                if (score > avg){
                    aboveAvgCount++;
                }
            }

            double percentage = ((double) aboveAvgCount / n) * 100;

            System.out.printf("%.3f%%%n", percentage);
        }
    }
}
